packages/forms/src/Testing/TestsForms.php

```diff
@@ -401,4 +402,111 @@ public function assertFormFieldIsVisible(): Closure
             return $this;
         };
     }
+
+    public function assertWizardStepExists(): Closure
+    {
+        return function (int $step, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormExists($formName);
+
+            /** @var ComponentContainer $form */
+            $form = $this->instance()->{$formName};
+
+            /**
+             * @var Wizard $wizard
+             */
+            $wizard = $form->getComponent(fn ($component) => $component instanceof Wizard);
+            Assert::assertTrue(count($wizard->getChildComponents()) >= $step, "Wizard does not have a step $step");
+
+            return $this;
+        };
+    }
+
+    public function assertWizardCurrentStepIs(): Closure
+    {
+        return function (int $step, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormExists($formName);
+
+            /** @var ComponentContainer $form */
+            $form = $this->instance()->{$formName};
+
+            /**
+             * @var Wizard $wizard
+             */
+            $wizard = $form->getComponent(fn ($component) => $component instanceof Wizard);
+            Assert::assertEquals(
+                $step,
+                $current = $wizard->getCurrentStepIndex() + 1,
+                "Failed asserting that wizard is on step $step, current step is {$current}"
+            );
+
+            return $this;
+        };
+    }
+
+    public function goToWizardStep(): Closure
+    {
+        return function (int $desiredStep = 1, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertWizardStepExists($desiredStep, $formName);
+
+            /** @var ComponentContainer $form */
+            $form = $this->instance()->{$formName};
+
+            /**
+             * @var Wizard $wizardComponent
+             */
+            $wizardComponent = $form->getComponent(fn ($component) => $component instanceof Wizard);
+            if ($desiredStep === -1) {
+                $desiredStep = count($wizardComponent->getChildComponents()) + 1;
```

Comment:
It takes you to the last form step, just wanted to cover the option since I was on it already, doesn't seem so useful now, I will remove it and maybe submit again in the future if I feel it is a missing functionality